scripts/port_status.py

```diff
@@ -13,6 +13,9 @@ def mk_label(path: Path) -> str:
     rel = path.relative_to(Path('src'))
     return str(rel.with_suffix('')).replace(os.sep, '.')
 
+def yaml_md_load(wikicontent: bytes):
```

Comment:
```suggestion
# So that the port status wiki page is human readable, we enclose the YAML with backticks,
# which need to be removed here.
def yaml_md_load(wikicontent: bytes):
```